.github/workflows/artifacts.yml

```diff
@@ -68,39 +68,78 @@ jobs:
           sed -i -e "/HTTPS_VERSION/s/[0-9]*\.[0-9]*\.[0-9]*/"${HTTPS_VERSION}"/" .dappnode_profile
           cat .dappnode_profile
 
-      # ISO ATTENDED
-      - name: Build attended
+      # Debian ISO ATTENDED
+      - name: Build Debian attended
         run: |
+          sed -i -e "/BASE_OS/s/ubuntu/debian/" docker-compose.yml
           sed -i -e "/UNATTENDED/s/true/false/" docker-compose.yml 
-          docker-compose build
-          docker-compose up
+          docker-compose up --build
```

Comment:
we should consider using new `docker compose` instead of legacy `docker-compose`